testdata/java-xref/obj/api/com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechRequest.Builder.yml

```diff
@@ -0,0 +1,20 @@
+### YamlMime:ManagedReference
+items:
+- uid: "com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechRequest.Builder"
+  id: "Builder"
+  parent: "com.google.cloud.mediatranslation.v1beta1"
+  langs:
+  - "java"
+  name: "StreamingTranslateSpeechRequest.Builder"
+  nameWithType: "StreamingTranslateSpeechRequest.Builder"
+  fullName: "com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechRequest.Builder"
+  type: "Class"
+  package: "com.google.cloud.mediatranslation.v1beta1"
+  summary: "Testing xrefs"
+  syntax:
+    content: "public static final class StreamingTranslateSpeechRequest.Builder extends GeneratedMessageV3.Builder<StreamingTranslateSpeechRequest.Builder> implements StreamingTranslateSpeechRequestOrBuilder"
+  inheritance:
+  - "java.lang.Object"
+  - "com.google.protobuf.AbstractMessageLite.Builder"
+  - "com.google.protobuf.AbstractMessage.Builder"
+  - "com.google.protobuf.GeneratedMessageV3.Builder"
```

Comment:
I think it might be missing a section for its own intro like in other [Java yml file](https://github.com/googleapis/doc-templates/blob/master/testdata/java/obj/api/com.google.cloud.speech.v1.RecognitionAudio.Builder.yml#L1195-L1198)? It's a replica of what's in xref.yml but perhaps it's because it's also missing in the yml file here?

Reply:
The other two xrefs work without that section for the test I added. So, I don't think it's that? I could be wrong.